plugin/build.gradle.kts

```diff
@@ -57,12 +58,14 @@ shadowJar.configure {
     }
 }
 
-tasks.getByName("jar").enabled = false
-tasks.getByName("jar").dependsOn(shadowJar)
+tasks.getByName("jar").apply {
```

Comment:
🚀 `tasks.jar { ... }` would be simpler and defer the configuration block

Reply:
Done, thanks